descriptors/ModuleDescriptor-template.json

```diff
@@ -190,6 +190,27 @@
         }
       ]
     },
+    {
+      "id": "voucher-storage.batch-vouchers",
```

Comment:
Interface should have a name `batch-voucher-storage.batch-vouchers` according to documentation.